crates/rooch-integration-test-runner/src/lib.rs

```diff
@@ -68,10 +69,10 @@ impl<'a> MoveTestAdapter<'a> for MoveOSTestAdapter<'a> {
 
     fn init(
         default_syntax: SyntaxChoice,
-        pre_compiled_deps: Option<&'a FullyCompiledProgram>,
-        task_opt: Option<TaskInput<(InitCommand, Self::ExtraInitArgs)>>,
+        option: Option<&'a FullyCompiledProgram>,
```

Comment:
`option` is not a good name. How about keeping the `pre_compiled_deps`?

Reply:
Alright, it's possible that I made a mistake and overlooked it.